test/common.go

```diff
@@ -5,15 +5,16 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-const maxValidNameLength = 62
+const maxValidNameLength = 63
 
-var TestOversizedNameObj = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+var TestOversizedNameObj = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijkl" // len=64 chars
```

Comment:
you could use `strings.Repeat("a", 64)` to obtain a variable of the desired length. Easier to maintain :)

Reply:
@xcoulon thanks for review.  Yup this make it maintainable but the same string is used at https://github.com/fabric8-services/fabric8-wit/pull/1878/files#diff-f420a1d6419315d243752d7e5b17d14e to validate test result.  In case test fail, with hardcoded string it's easy to find where in code this string is defined.  So I guess for better trouble-shooting for failed test, hard-coding as above is better :-)

Reply:
ok, fair point.